### PR TITLE
[capi] Rev versions

### PR DESCRIPTION
**Pull request by detiber**
- Update containerd to v1.2.9
- Update kubernetes to v1.15.4
- Fix image naming for CAPI AMIs

### CODE REVIEW

File: images/capi/packer/ami/packer.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
         "owners": ["099720109477"],
         "most_recent": true
       },
-      "ami_name": "capa-ami-ubuntu-18.04-{{user `kubernetes_semver`}}-{{user `build_timestamp`}}",
+      "ami_name": "capa-ami-ubuntu-18.04-{{user `kubernetes_deb_version`}}-{{user `build_timestamp`}}",
```
Comment (ncdc) [inline images/capi/packer/ami/packer.json]: These changes to use kubernetes_deb_version don't look right?

Comment (detiber) [reply images/capi/packer/ami/packer.json]: This is the pattern that we use for the lookups, this used to be called `kubernetes_version` and we had a filter to switch between deb and rpm version formats.

Comment (ncdc) [reply images/capi/packer/ami/packer.json]: But this means the name will have the `-00` at the end. I don't think that's necessary, is it?

Comment (ncdc) [reply images/capi/packer/ami/packer.json]: Oh, I see the lookup code expects/requires the kube semver followed by `-??`. Yuck.

Comment (detiber) [reply images/capi/packer/ami/packer.json]: Yeah, probably something that we want to clean up down the line, but would be quite a disruptive change now.

**Comment by codenrhoden** [comment]
/lgtm

I'll hold off for a bit in case there is additional feedback?

**Comment by ncdc** [comment]
I'm good to go

**Pull request merged**